images/linux/scripts/installers/homebrew.sh

```diff
@@ -18,6 +18,8 @@ brew shellenv|grep 'export HOMEBREW'|sed -E 's/^export (.*);$/\1/' | sudo tee -a
 # add brew executables locations to PATH
 brew_path=$(brew shellenv|grep  '^export PATH' |sed -E 's/^export PATH="([^$]+)\$.*/\1/')
 prependEtcEnvironmentPath "$brew_path"
+setEtcEnvironmentVariable HOMEBREW_NO_AUTO_UPDATE 1
+setEtcEnvironmentVariable HOMEBREW_NO_INSTALL_CLEANUP 1
```

Comment:
@MikeMcQuaid recommended avoiding HOMEBREW_NO_INSTALL_CLEANUP because of possible incorrect linkage

Reply:
Thanks both, agreed! I'd also perhaps suggest setting `HOMEBREW_AUTO_UPDATE_SECS` to a high value (as the check is very quick there)

Reply:
@MikeMcQuaid does not HOMEBREW_AUTO_UPDATE_SECS have the very same meaning as HOMEBREW_NO_AUTO_UPDATE does?

Reply:
`HOMEBREW_NO_AUTO_UPDATE` means it'll never check for auto updates, ever. `HOMEBREW_AUTO_UPDATE_SECS` means it'll only check for updates if it's been that long since it was updated. If set to e.g. 1 year that might be preferable, I dunno.

Reply:
@MikeMcQuaid we update the images much more often than once per year and catch up with the recent brew, so there's no sense to use HOMEBREW_AUTO_UPDATE_SECS

Reply:
Fine with me 👍🏻